src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -964,8 +966,9 @@ protected override Collection<PSDriveInfo> InitializeDefaultDrives()
                             }
                         }
 
-                        if (skipDuplicate)
+                        if (skipDuplicate) {
```

Comment:
Please move brace to new line. Below too.
```suggestion
                        if (skipDuplicate)
                        {
```